api/src/controllers/programs.ts

```diff
@@ -39,6 +39,11 @@ const zodMajorSpecPairSchema = z.object({
   ),
 });
 
+const MinorProgramSchema = z.object({
```

Comment:
Naming should be consistent with the Majors/Spec pairs

```suggestion
const zodMinorProgramSchema = z.object({
```

Alternatively, you can change `zodMajorSpecPairSchema` to be `majorSpecPairSchema` and make the first letter of `MinorProgramSchema` lowercase.